install/source/dependencies/index.markdown

```diff
@@ -58,10 +58,6 @@ Now you should be able to build using regular ``catkin build``
 
 OMPL is MoveIt!'s main planning library, their official install instructions can be found [here](http://ompl.kavrakilab.org/installation.html).
 
-To install from source within catkin, first ensure you do not have the binary installed versions, as it will override any local build of OMPL that your try to overlay. To understand the limitation see this [Github issue](https://github.com/ros-planning/moveit/issues/169#issuecomment-242849008):
-
-    sudo apt-get -qq remove ros-kinetic-ompl
```

Comment:
i don't understand - this is still required until https://github.com/ros-planning/moveit/issues/169 is resolved

Reply:
No, that's the beauty of the proposed description. :)
You "only" have to follow the new additional instructions below to exchange the includes in `moveit/moveit_planners/ompl/CMakeLists.txt`